Documentation/PlatformSupport.md

```diff
@@ -0,0 +1,69 @@
+
+# Platform Support
+
+Darwin platforms are binary and source stable. Linux platforms follow semantic versioning. Windows follows semantic versioning.
+
+### Windows concerns
+
+Trying to perfectly unify Windows with Darwin and Linux at the API level is ultimately a futile endeavor. However, in some specific cases, Windows does have directly corresponding syscalls or near-equivalents.
+
+Windows-specific details are open to reevaluation pending more Windows expertise for the project.
+
+
+## API to syscall mapping
+
+### `Errno`
+
+#### Instance computed properties
+
+| API                | Darwin     | Linux      | Windows    |
+|:-------------------|:-----------|:-----------|:-----------|
+| `description`      | `strerror` | `strerror` | `strerror` |
+| `debugDescription` | `strerror` | `strerror` | `strerror` |
+
+
+### `FileDescriptor`
+
+#### Instance methods
+
+| API                          | Darwin      | Linux       | Windows                                            |
+|:-----------------------------|:------------|:------------|:---------------------------------------------------|
+| `open`                       | `open`      | `open`      | `_wsopen_s(..., _SH_DENYNO, _S_IREAD | _S_IWRITE)` |
```

Comment:
The Windows entry doesn't seem to render properly.

Reply:
Backslash escaping it works for GH, though other renderers (including github.dev's renderer) will render the backslash.